components/Layout.tsx

```diff
@@ -245,7 +245,7 @@ const Footer = () => (
             Slack</a>
         </div>
         <div className='mb-4 mr-4'>
-          <a href='https://twitter.com/jsonschema' className='flex items-center text-white'><img src='/img/logos/twitter_logo-white.svg' className='w-4 h-4 mr-2' />
+          <a href='https://twitter.com/jsonschema' className='flex items-center text-white'><img src='/img/logos/x-twitter.svg' className='w-4 h-4 mr-2' />
```

Comment:
Why changing the Twitter logo in this PR? I think is better to stick to the PR purpose and avoid adding additional unrelated features.

There is an issue for this: #331 and this will be managed in other PR.

Reply:
I have made the required changes in the PR : )